src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -2041,12 +2039,11 @@ public PSCredential RunAsCredential
         internal PSCredential runAsCredential;
         internal bool isRunAsCredentialSpecified;
 
-#if !CORECLR // No ApartmentState In CoreCLR
         /// <summary>
         /// ApartmentState of the Runspace created for the shell.
         /// </summary>
         [Parameter()]
-        public ApartmentState ThreadApartmentState
+        public System.Threading.ApartmentState ThreadApartmentState
```

Comment:
Not possible to add `using System.Threading;` as in other files?

Reply:
`using System.Threading` is already there.  The compiler is complaining it can't find ApartmentState and not sure why that is.

Reply:
Hmmm, maybe I was working too late at the time or something, but I'm sure it was there and was not compiling, but looking at it now it isn't there...

Reply:
Type name collision with something else perhaps? 🤔